content/posts/por-que-usar-cdn-em-seu-site.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Por que você deveria usar uma CDN em seu site
```

Comment:
Esse título é uma pergunta? Se for, faltou o ponto de interrogação.
Caso não seja, ficaria melhor "Você deve usar uma CDN em seu site".
Aliás a palavra "deve" fica melhor que "deveria".

Reply:
Olá, @TailoMateus, obrigado pelos comentários.
O título não é uma pergunta. No caso, o "por que" tem o sentido de "razão" ou "motivo". Então o significado do título seria algo como "o motivo pelo qual você deveria usar uma CDN em seu site". Tenho dúvidas se devo ser tão afirmativo ao indicar "você deve usar uma CDN", afinal o recurso pode não ser aplicável em todas as situações. Acho que especificamente nesse caso, "deveria" desperta a curiosidade do leitor sem afirmar que todos devem usar. O que acha? Se tiver outra sugestão, sem problemas.

Reply:
Olá, se vai manter a palavra, 'Porque' no sentido de resposta precisa ser junto.

Reply:
Olá, obrigado pela sugestão. Essa é uma dúvida que sempre tenho, graças à complexidade da nossa língua portuguesa... rs
Nesse caso, acredito que podemos usar o por que separado. Segundo o professor Pasquale, o por que separado também pode ser usado em frases que não terminam com um ponto de interrogação. Veja: http://www.bbc.com/portuguese/brasil-39572054
Repare que o G1 usa títulos com o "por que" separado e com este sentido de "motivo": https://g1.globo.com/economia/tecnologia/noticia/por-que-um-premio-nobel-de-economia-acredita-que-o-bitcoin-deve-ser-proibido.ghtml

Reply:
Por mim bem tranquilo, dessa forma.